R/utils-fs.R

```diff
@@ -95,6 +95,11 @@ path_package_pkgdown <- function(package, ...) {
   }
 
   if (!file.exists(pkg_path)) {
+    # fall back on the bare templates directory
+    if (grepl("templatesBS.", paste0(...))) {
```

Comment:
I really don't like using grepl here. Could we make the bootstrap version an explicit argument to `path_package_pkgdown()` or otherwise handle upstream?